src/com/xilinx/rapidwright/util/RelocateModulesIntoBlackboxes.java

```diff
@@ -51,7 +51,6 @@
  */
 public class RelocateModulesIntoBlackboxes {
```

Comment:
It seems like only one `Module` is involved in this process.  Ideally, the name would be something like `RelocatingModuleBlackboxPopulator`.

Reply:
@eddieh-xlnx If you agree with the name Chris suggested, I will change it.

Reply:
OK from me.

Reply:
After I put it in, I see that it is very very long.      Do we need anything more than "BlackboxesPopulator"?    Will that confuse with any other feature/class in RW?

Reply:
changed to BlackboxesPopulator

Reply:
Sorry, I missed your last comment, how about `BlackboxPopulator`?

Reply:
Looks like there's a method called `DesignTools.populateBlackBox()` too?

Reply:
DesignTools.populateBlackBox() fills a blackbox from a given implementation.  This method fill multiple blackboxes from one implementation.  Thus, "es" is important.   If BlackboxesPopulator does not emphasize difference from DesignTools.populateBlackBox(), I can do MultipleBlackBoxesPopulator.             BTW, BlackBoxes or Blackboxes ?